new/LIT7077QeneNegusa.xml

```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7077QeneNegusa" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ንጉሠ፡ ነገሥት፡ ወልድ፡ አመ፡ ቆመ፡ በዓውደ፡ ተሰይጠ፡ ሰላሳ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Nǝguśa nagaśt wald ʾama qoma ba-ʿawda tasayṭa salāsā (Qǝne of kəbr yəʾəti-type)</title>
```

Comment:
Even if it is grammatically correct, the manuscript has በዓውደ, so it should be transcribed ba-ʿāwda (not ba-ʿawda).

Reply:
You are right! Thank you